docs/TypeScript.md

```diff
@@ -107,6 +107,8 @@ const config: WebdriverIO.Config = {
 export { config }
 ```
 
+If you are using this approach for a typed configuration, you have to remove the line with 'ts-node/register' from your framework options in your config file.
```

Comment:
Do we still need this one?

Reply:
Yes, if i don't remove this i have several ts compilation errors in combination with the preview of the javascript debugger